source/_integrations/evohome.markdown

```diff
@@ -189,7 +189,7 @@ For the location (controller), see `system_mode_status`:
 
 {% raw %}
 
-```text
+```jinja
```

Comment:
![CleanShot 2024-01-14 at 11 15 20@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/9bb1db22-f32d-4314-8b6e-97076156a5a6)

There is no difference...

Reply:
It seemed to on other usages of it in the repo, hence why I changed it.

Will have a look later when I'm at my computer; search on to the mobile app is pants.

Reply:
Or so I thought...

https://github.com/home-assistant/home-assistant.io/blob/bb4a912043f8779046e5b28304694f66c09da94d/source/_integrations/habitica.markdown?plain=1#L155-L161 does show some (very basic) highlighting (which I know, isn't the desired goal)

https://www.home-assistant.io/integrations/habitica#templating does not

But we do use `jinja` elsewhere, so at least this makes it more consistent, while the description in the PR is wrong about adding syntax highlighting (which it doesn't).

Obviously would be nice if the platform did support it.